### ISSUE TITLE
Crashing in immersive engineering manual

### ISSUE THREAD

**Issue by Aciho** [imrgr/rise-of-the-pack#15]
I get constant crashes with the immersive engineering manual open. I've had it while clicking on the controls on the multiblock building screens, or just when looking at normal pages.

Log:
https://pastebin.com/dKZsXx5J

**Comment by Watschman** [comment]
Is that the crashlog? Because it doesn't look like a normal minecraft crashlog to me, just a java process log which isn't really helpful unless your problem is not with minecraft but with java..

**Comment by Aciho** [comment]
Yeah, it's almost certainly the wrong log, it's been a while since I did this :) . I'll have a look tonight and see if I can find something more useful.

**Comment by Aciho** [comment]
OK, I tried again but unfortunately it crashes without leaving a log.

**Comment by Thornstromb** [comment]
Note since I'm wandering through: this bug is innate to IE itself, and is not the pack makers fault. i'm not currently sure what the explanation on this one is, as the "reason" has varied. once it was bad formatting on a specific page, then at one point it was pinned as being caused by Curse/Twitch using an outdated version of prepackaged java to run it's MC profiles. the IE devs are not being very responsive on this issue (possibly because it keeps reappearing when it shouldn't). regardless, it is quite annoying. Try using the manual in short bursts.

Imrgr - try working Immersive Petroleum into the pack. aside from the distillation tower/pumpjack being a massive beast to construct, and synergizing well with TE dynamos later on (distill tower can pump both Gasoline and Diesel into TE Compression Dynamos with coolant augment for really good fuel economy, or the pumpjack itself can go into a pair of Fractioning Stills to make Refined Fuel for a Plug-based Dynamo (or more. haven't done the math on the exact usage rate of X dynamos with X configurations running off the 6mb/t drip from the jack)
mostly though, IP adds a easily-made device which can be used to project build guides for IE multiblocks into the world, which makes IE dramatically easier to use.... and means you don't need to look in the book as much.
